Tests/EventListener/ParamConverterListenerTest.php

```diff
@@ -57,6 +57,41 @@ public function testAutoConvert($controllerCallable)
     }
 
     /**
+     * @dataProvider settingOptionalParamProvider
+     * @requires PHP 7.1
+     */
+    public function testSettingOptionalParam($function, $isOptional)
+    {
+        $kernel = $this->getMock('Symfony\Component\HttpKernel\HttpKernelInterface');
+        $request = new Request(array(), array(), array());
```

Comment:
If you're passing 3 empty arrays, you might as well omit them as all of them are optional.